templates/dependencies.html

```diff
@@ -0,0 +1,42 @@
+<div class="bower-config-file">
```

Comment:
This file does no longer exists... Please remove it.
This could be a result of a bad merge.